File: internal/model/article.go
package model

type Article struct {
	BaseModel

	Title   string `json:"title"`
	Slug    string `json:"slug"`
	Desc    string `json:"desc"`
	Cover   *string `json:"cover"`
	Content string `json:"content"`
	Draft   *bool  `json:"draft"`
	AiSummary *string `json:"ai_summary"`
}

File: packages/themekit/src/types/article.ts
export type Article = {
  // 文章id
  id: number;

  // 文章创建时间
  created_at: string;

  // 文章更新时间
  updated_at: string;

  // 文章slug
  slug: string;

  // 文章标题
  title: string;

  // 文章描述
  desc: string;

  // 文章内容
  content: string;
};

export type ArticleList = {
  // 文章总数
  count: number;

  // 文章列表
  articles: Article[];
};
